modules/mod_ginger_edit/templates/_action_ginger_dialog_new_rsc_tab.tpl

```diff
@@ -1,5 +1,5 @@
 {% with redirect|default:(dispatch=="ginger_edit")|if:[]:`ginger_edit` as redirect %}
-{% with callback|default:(dispatch=="ginger_edit")|if:"zAdminConnectDone":"" as callback %}
+{% with callback|default:"" as callback %}
```

Comment:
Is this backwards compatible in the situation where no `callback` argument is passed (which before defaulted to `ginger_edit`)?